promql/quantile.go

```diff
@@ -411,11 +417,20 @@ func ensureMonotonicAndIgnoreSmallDeltas(buckets buckets, tolerance float64) (bo
 			// Do not update the 'prev' value as we are ignoring the decrease.
 			buckets[i].count = prev
 			forcedMonotonic = true
+			if buckets[i].upperBound < forcedMonotonicMinBucket {
+				forcedMonotonicMinBucket = buckets[i].upperBound
+			}
+			if buckets[i].upperBound > forcedMonotonicMaxBucket {
+				forcedMonotonicMaxBucket = buckets[i].upperBound
+			}
+			if diff := prev - curr; diff > forcedMonotonicMaxDiff {
+				forcedMonotonicMaxDiff = diff
```

Comment:
I think showing non consequtive `le` values in the info can be confusing, so I'd store here the upperBounds of prev and curr and return that. `forcedMonotonicMinBucket` -> `forcedMonotonicLower`,  `forcedMonotonicMaxBucket` -> `forcedMonotonicUpper`